### DISCUSSION TITLE
Wails performance and build size benchmarks

### DISCUSSION THREAD

**Discussion by irfanbacker** [wailsapp/wails#1455] [General]
Hey. I just wanted to know if there are any benchmarks that I can refer to for wails. For someone who might have to decide to use something like electron, tauri or wails, having benchmarks may make it easier to make decisions. I am not able to find any details regarding performance and build size.

**Comment by leaanthony** [comment]
Hi 👋 Benchmarks are very subjective so there's little benefit to publishing any. The main reason for choosing Wails would be that Go is your preferred language. Good luck in your testing and let us know if you have any other questions 👍

**Reply by irfanbacker** [reply]
Maybe for direct comparisons, yes. Currently, I have no idea if a simple wails app would be of size 50MB OR 100MB OR 200MB. I think something like this would be a vital information for someone to even give it a try. I think people would hesitate to put in the time and effort to setup and run wails just to know this info(Certainly me). Im not asking for a thorough benchmark or comparison. Maybe something like an example app with it's build size and memory usage would suffice.

**Reply by leaanthony** [reply]
Yeah, we could publish the stats of the default projects (disk size, memory size). Where would you expect to see that information?

**Reply by irfanbacker** [reply]
Ideally, somewhere it is easy to notice for people taking a first look at the project. Somewhere in the github readme as well as the Introduction part of the docs maybe?
